.circleci/config.yml

```
@@ -86,7 +86,13 @@ jobs:
             mkdir truffle_workaround
             mv protocol truffle_workaround/
             cd truffle_workaround/protocol
-            yarn run test-concurrent
+            export PREV_SHA=$(git log HEAD~1 --pretty=format:%H -1)
```

nit: do you need export?
```suggestion
            PREV_SHA=$(git log HEAD~1 --pretty=format:%H -1)
```